$OpenBSD: patch-engine_core_ext_glee_GLee_h,v 1.1 2019/01/31 15:13:22 naddy Exp $

Mesa >= 18.2.5 has newer OpenGL headers from Khronos which change the
include guards.

Index: engine/core/ext/glee/GLee.h
--- engine/core/ext/glee/GLee.h.orig
+++ engine/core/ext/glee/GLee.h
@@ -63,7 +63,9 @@
 	#include <OpenGL/gl.h>
 #else // GLX
 	#define __glext_h_  /* prevent glext.h from being included  */
+	#define __gl_glext_h_  /* prevent glext.h from being included  */
 	#define __glxext_h_ /* prevent glxext.h from being included */
+	#define __glx_glxext_h_ /* prevent glxext.h from being included */
 	#define GLX_GLXEXT_PROTOTYPES
 	#include <GL/gl.h>
 	#include <GL/glx.h>
